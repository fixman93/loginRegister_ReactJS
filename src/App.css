.nav ul {
  max-width: 300px;
  display: flex;
  justify-content: space-between;
  margin: auto;
  border: 1px solid #999;
  padding: 0;
  list-style: none;
}

.nav ul li {
  border-right: 1px solid #999;
  width: 50%;
  padding: 10px;
  text-align: center;
  cursor: pointer;
}

.nav ul li:last-child {
  border: none;
}

.form {
  max-width: 300px;
  margin: 30px auto;
  border: 1px solid #999;
  padding: 20px 10px;
  text-align: center;
}

.form input {
  width: 100%;
  padding: 5px;
  border: 0;
  margin-bottom: 15px;
  min-height: 50px;
  border-radius: 5px;
}

* {
  box-sizing: border-box;
}

.submit-btn, .register-btn {
  display: block;
  width: 100%;
  background: #ccc;
  text-align: center;
  line-height: 50px;
  border-radius: 5px;
  color: white;
  font-weight: bold;
  text-transform: uppercase;
  cursor: pointer;
  font-size: 16px;
}

button.register-btn {
  box-shadow: none;
  outline: none;
}

.nav ul li.active {
  background: gray;
}