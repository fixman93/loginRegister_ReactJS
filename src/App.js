import React, { Component } from 'react';
import { Switch, Route } from 'react-router-dom'


import Home from './home'


class App extends Component {
  render() {
    return (
      <div>
        <Switch>
          <Route exact path='/' component={Home} />
        </Switch>
      </div>
    );
  }
}

export default App;
