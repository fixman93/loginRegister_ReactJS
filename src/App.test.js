import React from 'react'
import Adapter from 'enzyme-adapter-react-16'
import { shallow, configure } from 'enzyme'
import App from './App'


configure({ adapter: new Adapter() })

describe('Render App', () => {
  const mockLogin = jest.fn()
  const props = { loginSubmit: mockLogin }
  const app = shallow(<App {...props} />)
  it('render app properly', () => {
    expect(app).toMatchSnapshot()
  })

  it('find login/register form', () => {
    expect(app.find('Login').exists()).toBe(true)
  })

})