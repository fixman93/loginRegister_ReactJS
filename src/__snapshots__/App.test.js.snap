// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render App snapshot app page 1`] = `ShallowWrapper {}`;
